test/lib/y2users/linux/writer_test.rb

```diff
@@ -51,6 +51,39 @@
 
     let(:username) { "testuser" }
     let(:pwd_value) { Y2Users::PasswordEncryptedValue.new("$6$3HkB4uLKri75$Qg6Pp") }
+    let(:keyring) { double(Yast::Users::SSHAuthorizedKeyring, write_keys: true) }
+
+    before do
+      allow(Yast::Execute).to receive(:on_target!)
+      allow(Yast::Users::SSHAuthorizedKeyring).to receive(:new).and_return(keyring)
+    end
+
+    RSpec.shared_examples "writing authorized keys" do
+      let(:home) { "/home/y2test" }
+
+      before do
+        current_user = config.users.find { |u| u.id == user.id }
```

Comment:
NP: we can use the new API here:

~~~
current_user = config.users.by_id(user.id)
~~~

Reply:
Right, I didn't realize it when rebasing changes